jbehave-support-core/src/main/java/org/jbehavesupport/core/healthcheck/HealthCheckSteps.java

```diff
@@ -48,26 +45,23 @@ public final class HealthCheckSteps {
     private final ConfigurableListableBeanFactory beanFactory;
 
     @Given("these components are healthy:$componentList")
-    public void checkComponentsAreHealthy(ExamplesTable componentList) throws MultipleFailureException {
+    public void checkComponentsAreHealthy(ExamplesTable componentList) {
         List<String> componentQualifiers = componentList.getRowsAsParameters().stream()
             .map(Row::values)
             .map(row -> row.get("component"))
-            .collect(toList());
+            .toList();
 
-        List<Throwable> exceptions = new ArrayList<>();
+        SoftAssertions softly = new SoftAssertions();
         for (String componentQualifier : componentQualifiers) {
             HealthCheck healthCheck = resolveHealthCheck(componentQualifier);
             try {
                 healthCheck.check();
             } catch (Exception e) {
-                IllegalStateException ie = new IllegalStateException("Component " + componentQualifier + " is not healthy. " + e.getMessage(), e);
-                exceptions.add(ie);
+                softly.fail("Component " + componentQualifier + " is not healthy. " + e.getMessage(), e);
             }
         }
 
-        if (!exceptions.isEmpty()) {
-            throw new MultipleFailureException(exceptions);
```

Comment:
We should probably mention this incompatible API change somewhere, but dont have any idea where... Apart of release notes when releasing new version I cant think of anything else - any other ideas or are release notes enough?